src/node_crypto.h

```diff
@@ -259,6 +267,17 @@ class SSLWrap {
                                      unsigned int inlen,
                                      void* arg);
 #endif  // OPENSSL_NPN_NEGOTIATED
+#ifdef TLSEXT_TYPE_application_layer_protocol_negotiation
+  static void GetALPNNegotiatedProto(
+    const v8::FunctionCallbackInfo<v8::Value>& args);
```

Comment:
Yes, I've tested and this causes compile error. #ifdef  is removed in here and wrong wrapping indents are also fixed. #ifdef in SetMethod is also removed for the functions to return undefined.